packages/broker/develop.md

```diff
@@ -0,0 +1,82 @@
+
+## Develop
+
+The information here is mostly for the Streamr team and others intending to contribute to the codebase.
+
+When developing the Broker, it is convenient to run it as part of the full Streamr development stack. Check out
+the [streamr-docker-dev](https://github.com/streamr-dev/streamr-docker-dev) tool that can be used to run the full stack.
+
+If instead you want to run a broker node by itself without Docker, follow the steps in the [Run](#run) section.
+
+See folder "configs" for example configurations. To run a broker connected to the local dev environment:
+```
+broker configs/development-1.env.json
+```
+Then run the command tracker with default values
+```
+tracker
+```
+
+To run tests, first install dependencies:
+
+    npm ci
+
+Run the tests:
+
+    npm run test
+
+We use [eslint](https://github.com/eslint/eslint) for code formatting:
+
+    npm run eslint
+
+Code coverage:
+
+    npm run coverage
+
+### Debug
+
+To get all debug messages:
+
+    LOG_LEVEL=debug
+
+... or adjust debugging to desired level:
+
+    LOG_LEVEL=[debug|info|warn|error]
+
+To disable all logs
+
+    NOLOG=true
+
+
+## Release
```

Comment:
The up-to-date release procedure is described in monorepo root README.md: https://github.com/streamr-dev/network-monorepo/blob/main/README.md